app/javascript/chat/util.js

```diff
@@ -76,39 +81,41 @@ export function adjustTimestamp(timestamp) {
   return time;
 }
 
-
 export function setupNotifications() {
-  navigator.serviceWorker.ready.then((serviceWorkerRegistration) => {
-    serviceWorkerRegistration.pushManager.getSubscription()
-      .then(function(subscription) {
+  navigator.serviceWorker.ready.then(serviceWorkerRegistration => {
+    serviceWorkerRegistration.pushManager
+      .getSubscription()
+      .then(subscription => {
         if (subscription) {
           return subscription;
         }
         return serviceWorkerRegistration.pushManager.subscribe({
           userVisibleOnly: true,
-          applicationServerKey: window.vapidPublicKey
+          applicationServerKey: window.vapidPublicKey,
         });
-      }).then(function(subscription) {
-        sendKeys(subscription.toJSON(), null, null)
+      })
+      .then(subscription => {
+        sendKeys(subscription.toJSON(), null, null);
       });
   });
 }
 
 export function getNotificationState() {
-  //Not yet ready
-  if (!window.location.href.includes('ask-for-notifications')){
-    return "dont-ask"
+  // Not yet ready
```

Comment:
A little not for whoever comes across this PR, there are only three permissions available. See https://developer.mozilla.org/en-US/docs/Web/API/Notification/permission